### PR TITLE
add log for init TiSession

### PR DESCRIPTION
**Pull request by marsishandsome**
Signed-off-by: marsishandsome <marsishandsome@gmail.com>

### What problem does this PR solve? 

add log for init TiSession

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to be included in the release note

### CODE REVIEW

File: src/main/java/org/tikv/common/PDClient.java [hunk 1]
```diff
@@ -575,14 +576,18 @@ private void initCluster() {
...
+    logger.info("init host mapping: start");
     this.hostMapping =
         Optional.ofNullable(getConf().getHostMapping())
             .orElseGet(() -> new DefaultHostMapping(this.etcdClient, conf.getNetworkMappingName()));
+    logger.info("init host mapping: end");
     // The first request may cost too much latency
     long originTimeout = this.timeout;
     this.timeout = 2000;
     for (URI u : pdAddrs) {
+      logger.info("get members: start");
```
Comment (birdstorm) [inline src/main/java/org/tikv/common/PDClient.java:588]: add some description such as get pd members from <pd_address>?

File: src/main/java/org/tikv/common/PDClient.java [hunk 2]
```diff
@@ -603,7 +608,9 @@ private void initCluster() {
 
     String leaderUrlStr = resp.getLeader().getClientUrls(0);
     leaderUrlStr = uriToAddr(addrToUri(leaderUrlStr));
+    logger.info("createLeaderClientWrapper: start");
```
Comment (birdstorm) [inline src/main/java/org/tikv/common/PDClient.java:611]: ditto

**Review by birdstorm — APPROVED** [review]
rest LGTM

**Review by birdstorm — APPROVED** [review]
LGTM

File: src/main/java/org/tikv/common/PDClient.java [hunk 3]
```diff
@@ -575,14 +576,18 @@ private void initCluster() {
                         .setDaemon(true)
                         .build()))
             .build();
+    logger.info("init host mapping: start");
     this.hostMapping =
         Optional.ofNullable(getConf().getHostMapping())
             .orElseGet(() -> new DefaultHostMapping(this.etcdClient, conf.getNetworkMappingName()));
+    logger.info("init host mapping: end");
```
Comment (zz-jason) [inline src/main/java/org/tikv/common/PDClient.java:583]: it's better to aggregate these logs into one single log:

```
init host mapping: started in xxx, tasks 12ms(or 12s)
```

**Pull request merged**